### DISCUSSION TITLE
How do I view/load the USA bandplan?

### DISCUSSION THREAD

**Discussion by javierxio** [AlexandreRouma/SDRPlusPlus#1533] [Q&A] [answered]
Hello. I am using on Linux with RTL-SDR V4 dongle. 

How can I see the different bands? So far, I can see FM Broadcast, Air Band Voice, Marine, PMR446, etc.

But I would like to see FRS, all the TV channels, and NOAA Weather Radio as well. I suppose there is a setting that I need to select so the USA frequency spectrum can apply?

**Comment by KubaPro010** [comment, marked as answer]
on the left-side menu where you can scroll you have `Band Plan`, open that and select your band plan (really?)
